common/cpp/src/google_smart_card_common/value_conversion_unittest.cc

```diff
@@ -578,14 +578,14 @@ TEST(ValueConversion, UnsignedLongToValue) {
     EXPECT_TRUE(ConvertToValue(kZero, &value, &error_message));
     EXPECT_TRUE(error_message.empty());
     ASSERT_TRUE(value.is_integer());
-    EXPECT_EQ(value.GetInteger(), kZero);
+    EXPECT_EQ(value.GetInteger(), 0);
   }
 
   {
     Value value;
     EXPECT_TRUE(ConvertToValue(kNumber, &value));
     ASSERT_TRUE(value.is_integer());
-    EXPECT_EQ(value.GetInteger(), kNumber);
+    EXPECT_EQ(value.GetInteger(), static_cast<int64_t>(kNumber));
```

Comment:
Why are we casting kNumber to int64_t here, when we are casting the other argument in the other tests?

Reply:
These are opposite tests. Here `value.GetInteger()` is `int64_t` and `kNumber` is `unsigned long`. In the tests below `converted` is `int64_t` and `kNumber` is `int64_t`. What the commit does is casting them all to be `int64_t`.